diveMap/MapViewController.swift

```diff
@@ -65,7 +69,7 @@ class MapViewController: UIViewController, MKMapViewDelegate {
     }
     
     func setZoomInitialLocation(location: CLLocationCoordinate2D) {
-//        let currentLocation = CLLocationCoordinate2D.init(latitude: 47.6031537682643, longitude: -122.336164712906 )
+        //        let currentLocation = CLLocationCoordinate2D.init(latitude: 47.6031537682643, longitude: -122.336164712906 )
         let regionRadius : CLLocationDistance = 25
```

Comment:
remove unused code, maybe create a static property in your constants file for the distance instead of just haveing a number or create a properpty at top of the class